Products/ZenModel/ProductClass.py

```diff
@@ -52,8 +54,9 @@ class ProductClass(ZenModelRM, ZenPackable):
     )
 
     _relations = ZenPackable._relations + (
-        ("instances", ToMany(ToOne, "Products.ZenModel.MEProduct", "productClass")),
         ("manufacturer", ToOne(ToManyCont,"Products.ZenModel.Manufacturer","products")),
+        # Deprecated
```

Comment:
The migrate script drops the relationship, so you can just drop this. It isn't deprecated so much as gone.